src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
I have a problem with the doc type. We may want to refer how the PSR-5 is defining..

```php
function query1($hello, ...$args)
{
    var_dump($args);
}

query1('Hello', 'World', 'And');
query1('Hello', ['World', 'And']);
```

For the `$args` we are mentioning it is an array. ie correct, but will it confuse people and send an array as in the second example?

See https://3v4l.org/g8Wgm

I am not seeing anything quick on my search : https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md